spring-resttemplate-2/src/main/java/com/baeldung/resttemplate/json/service/UserService.java

```diff
@@ -0,0 +1,9 @@
+package com.baeldung.resttemplate.json.service;
+
+import com.baeldung.resttemplate.json.model.User;
+
+import java.util.List;
+
+public interface UserService {
+    List<User> getUsers() ;
+}
```

Comment:
While it may be good practice to extract interfaces from services, this interface seems to complicate this simple example. We could just as easily do without it.

Reply:
We don't need to include this bit in the article, but as you say it is good practice so I would prefer to leave it in unless you REALLY object it. :)